pennylane/ops/op_math/condition.py

```diff
@@ -140,14 +245,14 @@ def cond(condition, true_fn: Callable, false_fn: Optional[Callable] = None, elif
 
     Args:
         condition (Union[.MeasurementValue, bool]): a conditional expression involving a mid-circuit
-           measurement value (see :func:`.pennylane.measure`). This can only be of type ``bool`` when
-           decorated by :func:`~.qjit`.
+           measurement value (see :func:`.pennylane.measure`).

```

Comment:
This should also say booleans are allowed.

Reply:
a65934036

Reply:
[a659340](https://github.com/PennyLaneAI/pennylane/pull/6016/commits/a659340364e73569a67aed9db71cc7adca87fb8f)